### PR TITLE
add postgres package

### PR DESCRIPTION
**Pull request by rawlingsj**
Signed-off-by: James Rawlings <jrawlings@chainguard.dev>

### CODE REVIEW

**Comment by rawlingsj** [comment]
contributed by @rawlingsj @luhring @kaniini @patflynn @jvanzyl

**Comment by rawlingsj** [comment]
hmm, that CI build finished very quickly.  I don't think it actually built the postgres package

File: postgres/postgresql-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env bash
```
Comment (luhring) [inline postgres/postgresql-entrypoint.sh:1]: Would it be helpful at all to put a note at the top here about where this script came from / what it was based on?

Comment (rawlingsj) [reply postgres/postgresql-entrypoint.sh:1]: good call, will do that now

File: postgresql.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/postgresql/APKBUILD?h=3.3-stable
```
Comment (kaniini) [inline postgresql.yaml:1]: uhh :)

File: postgresql.yaml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+  - name: libpq
+    description: PostgreSQL libraries
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/lib
+          mv ${{targets.destdir}}/usr/lib/libpq.so* ${{targets.subpkgdir}}/usr/lib/
+
+  - name: postgresql-oci-entrypoint
+    description: PostgreSQL libraries
```
Comment (kaniini) [inline postgresql.yaml:85]:
```suggestion
    description: Entrypoint for using PostgreSQL in OCI containers
```

**Review by kaniini — CHANGES_REQUESTED** [review]

File: postgresql.yaml (hunk 2 above)
Comment (rawlingsj) [reply postgresql.yaml:1]: damn! removing now

**Review by kaniini — APPROVED** [review]

**Pull request merged**